boards/arm/sam_e70_xplained/doc/sam_e70_xplained.rst

```diff
@@ -141,6 +141,17 @@ Flashing
    The command will also verify that the image was programmed correctly, reset
    the board and run the Zephyr application.
 
+   It is also possible to use an external debug adapter such as J-Link or ULINK
+   connected to a 20-pin JTAG header to flash the image. In this case the name
+   of the debug adapter should be supplied via OPENOCD_INTERFACE variable.
+   OpenOCD will look for the appropriate interface configuration file in
+   interface/ folder on its internal search path. This is similar to a command
+   line -f interface/....cfg option.
```

Comment:
Maybe it would be better to write
_"-f interface/$OPENOCD_INTERFACE.cfg option."_
It's longer but more explicit?